features/admin_can_crud_article.feature

```diff
@@ -0,0 +1,47 @@
+Feature: Admin can add an article
+  As an Admin
+  In order to add articles on the website
+  I would like to get CRUD access on my dashboard
+
+  Background:
+    Given The following Admin exists
+      | email               | password  |
+      | admin@example.com   | password  |
+    Given The following Article exists
+      | title               | content   |
+      | MyString            | MyText    |
+    Given I am on the Admin login page
+    Given I fill in "admin_user_email" with "admin@example.com"
+    Given I fill in "admin_user_password" with "password"
```

Comment:
I would like you to avoid using snake case in acceptance tests. Can't you use natural language and transform the value to snake case in your step definition?